lib/mesa-lib/BUILD

```diff
@@ -9,6 +9,12 @@ else
   OPTS+=" --enable-egl --enable-osmesa --with-gallium-drivers="
 fi  &&
 
+OPTS+=" --with-egl-platforms=drm,x11" &&
+
+if module_installed wayland; then
+  OPTS+=",wayland"
```

Comment:
@Florin65 we should be using "if in_depends $MODULE wayland" instead, otherwise the optional_dependency is useless if you have wayland installed for something else :)